purview/base.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import json
+
+
+class BaseAPI(object):
+    """
+    Base API class to standardize the main API functionality
+    """
+
+    def serialize(self, node_obj):
+        """
+        Serializes a Node object to JSON
+        :param node_obj: a model object
+        :return: JSON string
+        """
+        return json.dumps(node_obj.__properties__, indent=4, sort_keys=True, default=str)
```

Comment:
This should return a dictionary instead of JSON so that way we can construct JSON with multiple nodes outside of this function.